pkg/koordlet/runtimehooks/hooks/extendedresource/extended_resource.go

```diff
@@ -379,6 +380,7 @@ func (p *plugin) SetContainerMemoryLimit(proto protocol.HooksProtocol) error {
 	return nil
 }
 
-func isPodQoSBEByAttr(labels map[string]string, annotations map[string]string) bool {
-	return apiext.GetQoSClassByAttrs(labels, annotations) == apiext.QoSBE
+func isPodQoSMatchedByAttr(labels map[string]string, annotations map[string]string) bool {
+	podQoS := apiext.GetQoSClassByAttrs(labels, annotations)
+	return podQoS == apiext.QoSBE || podQoS == apiext.QoSLS
```

Comment:
which cgroup will Mid+LS pods use
/besteffort may be not appropriate
more things need to be clarify if using /burstable, such as the fairness of cpushares value between Prodand Mid

Reply:
This function is just a filter which means it allow LS and BE.

Reply:
Mid+BE in /besteffort seems reasonable, while Mid+LS should better use /burstable(but this needs some hack works since kubelet is not allowed yet)

I think now we can let koordlet supports Mid+BE first.

So here are the things we should fix:
1. keep podQOSConditions still as apiext.QoSBE
2. define independent GetContainerMidMilliCPURequest and Memory instead of using GetContainerBatchMilliCPURequest

Moreover, there are several QoS policies for BE, we need to clarify whether they should support Mid+BE and how to support, for example the evict order of pods. it will be better if we have some docs first.